d2/src/test/java/com/linkedin/d2/balancer/strategies/random/RandomLoadBalancerTest.java

```diff
@@ -44,10 +44,13 @@ public void testRoundRobinBalancer()
     partitionDataMap.put(DefaultPartitionAccessor.DEFAULT_PARTITION_ID, new PartitionData(1d));
     TrackerClient trackerClient1 = Mockito.mock(TrackerClient.class);
     TrackerClient trackerClient2 = Mockito.mock(TrackerClient.class);
-    List<TrackerClient> trackerClients = new ArrayList<TrackerClient>();
+    Map<URI, TrackerClient> trackerClients = new HashMap<>();
 
-    trackerClients.add(trackerClient1);
-    trackerClients.add(trackerClient2);
+    URI uri1 = URI.create("http://cluster-1/test");
+    URI uri2 = URI.create("http://cluster-1/test");
```

Comment:
Do you set the same uri to 2 trackerClients on purpose? Should we explicitly verify it will always choose trackerClient2?

Reply:
"Do you set the same uri to 2 trackerClients on purpose?" For this test the URI value is irrelevant, but I can change to be different to avoid any confusion.
 
"Should we explicitly verify it will always choose trackerClient2?" this shouldn't be the case right since the algorithm should choose the client randomly.